src/app/app.component.ts

```diff
@@ -0,0 +1,111 @@
+import { HttpClient } from '@angular/common/http';
+import { AfterViewInit, Component, ViewChild } from '@angular/core';
+import { MatSort } from '@angular/material/sort';
+import { MatTable, MatTableDataSource } from '@angular/material/table';
+import { tap } from 'rxjs';
+import { FilterValue } from './filter/filter.component';
+import { typeColors } from './legend/legend.component';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent implements AfterViewInit {
+  private currentData: Row[] = [];
+  private filterValue: FilterValue = {
+    currentPage: 0,
+    entriesPerPage: 20
+  };
+
+  readonly title = 'BG Queue Abuser Viewer';
```

Comment:
I don't think it is really needed

Reply:
Added string directly in template and removed variable from component